code/modules/mob/living/carbon/human/species/necromorph/twitcher.dm

```diff
@@ -194,6 +194,8 @@ All of these properties combined make Step Strike tricky and disorienting to use
 /datum/species/necromorph/slasher/twitcher/handle_organ_external_damage(var/obj/item/organ/external/organ, brute, burn, damage_flags, used_weapon)
 	var/mob/living/L = organ.owner
 	var/datum/extension/twitch/T = get_extension(L, /datum/extension/twitch)
+	if(get_extension(L, /datum/extension/stasis_effect))
+		return
```

Comment:
are you sure a simple return is adequate here? there's quite a bit of complexity in how this works, iirc

Reply:
whoops, forgot to add `..()` after `return`